cmd/query/app/static_handler_test.go

```diff
@@ -51,3 +53,28 @@ func TestDefaultStaticAssetsRoot(t *testing.T) {
 	handler := NewStaticAssetsHandler("")
 	assert.Equal(t, "jaeger-ui-build/build/", handler.staticAssetsRoot)
 }
+
+func TestRegisterRoutesHandler(t *testing.T) {
+	r := mux.NewRouter()
+	handler := NewStaticAssetsHandler("fixture/")
+	handler.RegisterRoutes(r)
+	server := httptest.NewServer(r)
+	expectedRespString := "Test Favicon\n"
+	defer server.Close()
```

Comment:
nit: move defer one line up. Keep the initialization of the server and the closing of as close as possible.